-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 37.937 s <<< FAILURE! - in TestSuite
failing_it(package1.Test1)  Time elapsed: 0.019 s  <<< FAILURE!
java.lang.AssertionError: expected [2] but found [1]
	at package1.Test1.failing_it(Test1.java:31)

